code/AdobeEdge.brs

```diff
@@ -270,6 +270,13 @@ function AdobeSDKInit() as object
         _private: {
             ' constants
             cons: _adb_InternalConstants(),
+            ' for testing purpose
+            lastEventId: invalid,
+            ' for testing purpose
+            resetSDK: sub ()
```

Comment:
I hope customers don't use this.

Reply:
The `_private` object holds it, so it is recommended that customers avoid calling this kind of `internal` function. However, we can move the code to the test setup section as well. Inform me if you would like me to proceed with that.